tests/falkor_test.js

```diff
@@ -49,6 +49,20 @@ exports.testRequestFailures = function (test) {
 }
 
 
+// Tests that an SSL requests uses https
+exports.testSsl = function (test) {
+  var mockTest = newMockTestWithNoAssertions(test)
+
+  mockTest.verifyResponse(nock('https://falkor.fake:8080')
+      .get('/secure', undefined, {ca: 'CustomCA-123'})
```

Comment:
Is this a file we should check-in?

Reply:
This didn't really work. Nock didn't seem to have a way to filter on the options.ca field. I think I will just remove it